website/src/blogs/2018-08-29-openebs-0-7-release-pushes-cstor-storage-engine-to-field-trials.md

```diff
@@ -0,0 +1,186 @@
+    - name: ReplicaCount
+    value: "1"
+     - name: ReplicaAntiAffinityTopoKey
+     value: failure-domain.beta.kubernetes.io/zone
+    - name: StoragePool
+    value: default
+    provisioner: openebs.io/provisioner-iscsi
+    parameters:
+    openebs.io/fstype: "xfs"
+
+**Step 2:** Specify the application label (say *app-unique-id*) in the Mongo StatefulSet:
+
+    apiVersion: apps/v1beta1
+    kind: StatefulSet
+    metadata:
+    name: mongo
+    spec:
+    serviceName: "mongo"
+    replicas: 3
+    template:
+    metadata:
+    labels:
+    role: mongo
+    environment: test
+    openebs.io/replica-anti-affinity: app-unique-id
+
+**Step 3:** Specify the Storage Class in the Volume Claim Template:
+
+    volumeClaimTemplates:
+    — metadata:
+    name: mongo-persistent-storage
+    spec:
+    storageClassName: mongo-pv-az
+    accessModes:
+    — ReadWriteOnce
+    resources:
+    requests:
+    storage: 5G
+
+When the PVs are created for the Mongo StatefulSet, the Replicas are configured with PodAntiAffinity rules with the provided application label and topology key as follows:
+
+    podAntiAffinity:
+    requiredDuringSchedulingIgnoredDuringExecution:
+    — labelSelector:
+    matchLabels:
+    openebs.io/replica: jiva-replica
+    openebs.io/replica-anti-affinity: app-unique-id
+    topologyKey: failure-domain.beta.kubernetes.io/zone
+
+The complete YAML file for Mongo StatefulSet using OpenEBS PV is available [here](https://github.com/openebs/openebs/blob/master/k8s/demo/mongodb/mongo-statefulset.yml).
+
+**Example 2:** When running in hyper-converged mode along with applications, it is imperative to deploy granular control on how much memory or cpu can be allocated for serving a storage volume. Again, this is a relatively common scenario, and we know how to control it for the Application Deployments! The same can be extended to the Storage!
+
+For example, you can define the Resource Requests and Limits for different storage pods:
+
+    apiVersion: storage.k8s.io/v1
+    kind: StorageClass
+    metadata:
+    name: openebs-jiva-limits
+    annotations:
+    cas.openebs.io/config: |
+    - name: TargetResourceRequests
+    value: |-
+    memory: 0.5Gi
+    cpu: 100m
+    - name: TargetResourceLimits
+    value: |-
+    memory: 1Gi
+    - name: ReplicaResourceRequests
+    value: |-
+    memory: 0.5Gi
+    cpu: 100m
+    - name: ReplicaResourceLimits
+    value: |-
+    memory: 2Gi
+    cpu: 200m
+
+I have only covered a couple of examples here , but I am very excited that with the framework we have now embedded within OpenEBS, we can support any kind of scheduling requests you might have. **Bring it On!**
+
+—
+
+**cStor Storage Engine (cStor)** is the latest addition to the storage engines supported by OpenEBS. Similar to the previous Jiva/Longhorn storage engine, cStor is also completely Open Source and has no dependency on the Kernel. It can run in the platform of your choice — either on premise or cloud!
```

Comment:
```suggestion
**cStor Storage Engine (cStor)** is the latest addition to the storage engines supported by OpenEBS. Similar to the previous Jiva/Longhorn storage engine, cStor is also completely Open Source and has no dependency on the Kernel. It can run on the platform of your choice — either on-premise or cloud!
```